certified-connectors/Secib/apiDefinition.swagger.json

```diff
@@ -1173,6 +1353,234 @@
           }
         ]
       }
+    },
+    "/api/v1/secib/TempsPasses": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {}
```

Comment:
@matthieuHellaSecib empty schemas are not allowed. Please define a schema or remove it in case it's empty. More information about responses can be found in [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).